hooks.py

```diff
@@ -9,15 +10,14 @@ def set_locale():
         if app.config['FORCE_LANGUAGE']:
             g.CURRENT_LOCALE = app.config['FORCE_LANGUAGE']
         else:
-            g.CURRENT_LOCALE = request.accept_languages.best_match(app.config['LANGUAGES'].keys(), default=app.config['DEFAULT_LANGUAGE'])
+            g.CURRENT_LOCALE = request.accept_languages.best_match(app.config['LANGUAGES'].keys(),
+                                                                   default=app.config['DEFAULT_LANGUAGE'])
 
 
 @auth.get_password
 def get_password(username):
-    if username in app.config['USERS']:
-        return app.config['USERS'].get(username)
-
-    return None
+    backend = get_current_auth_backend()
+    return backend.retrieve_password(username)
```

Comment:
A little blank line before this return? :)

Reply:
Addressed in latest commits.